File: SurTerrain/src/Models/clubx.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Models;

/**
 *
 * @author Hamidou
 */
public class clubx {
    private int id_clubx,id,id_club;
    private String Nom_club,Nom_joueur;

    public clubx(int id_clubx, int id, int id_club, String Nom_club, String Nom_joueur) {
        this.id_clubx = id_clubx;
        this.id = id;
        this.id_club = id_club;
        this.Nom_club = Nom_club;
        this.Nom_joueur = Nom_joueur;
    }

    public clubx(int id, int id_club, String Nom_club, String Nom_joueur) {
        this.id = id;
        this.id_club = id_club;
        this.Nom_club = Nom_club;
        this.Nom_joueur = Nom_joueur;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getId_club() {
        return id_club;
    }

    public void setId_club(int id_club) {
        this.id_club = id_club;
    }

    public String getNom_club() {
        return Nom_club;
    }

    public void setNom_club(String Nom_club) {
        this.Nom_club = Nom_club;
    }

    public String getNom_joueur() {
        return Nom_joueur;
    }

    public void setNom_joueur(String Nom_joueur) {
        this.Nom_joueur = Nom_joueur;
    }
    
    
    
}

File: src/Entity/Clubx.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Clubx
 *
 * @ORM\Table(name="clubx", indexes={@ORM\Index(name="clubet", columns={"id_club"}), @ORM\Index(name="joueuret", columns={"id"})})
 * @ORM\Entity
 * @ORM\Entity(repositoryClass="App\Repository\ClubxRepository")
 */
class Clubx
{
    /**
     * @var int
     *
     * @ORM\Column(name="id_clubx", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idClubx;

    /**
     * @var string
     *
     * @ORM\Column(name="Nom_club", type="string", length=30, nullable=false)
     */
    private $nomClub;

    /**
     * @var string
     *
     * @ORM\Column(name="Nom_joueur", type="string", length=30, nullable=false)
     */
    private $nomJoueur;

    /**
     * @var \Club
     *
     * @ORM\ManyToOne(targetEntity="Club")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_club", referencedColumnName="id_club")
     * })
     */
    private $idClub;

    /**
     * @var \Joueurs
     *
     * @ORM\ManyToOne(targetEntity="Joueurs")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id", referencedColumnName="id")
     * })
     */
    private $id;

    public function getIdClubx(): ?int
    {
        return $this->idClubx;
    }

    public function getNomClub(): ?string
    {
        return $this->nomClub;
    }

    public function setNomClub(string $nomClub): self
    {
        $this->nomClub = $nomClub;

        return $this;
    }

    public function getNomJoueur(): ?string
    {
        return $this->nomJoueur;
    }

    public function setNomJoueur(string $nomJoueur): self
    {
        $this->nomJoueur = $nomJoueur;

        return $this;
    }

    public function getIdClub(): ?Club
    {
        return $this->idClub;
    }

    public function setIdClub(?Club $idClub): self
    {
        $this->idClub = $idClub;

        return $this;
    }

    public function getId(): ?Joueurs
    {
        return $this->id;
    }

    public function setId(?Joueurs $id): self
    {
        $this->id = $id;

        return $this;
    }


}
